apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -148,7 +143,7 @@ const ChangeProjectDomain = ({
 
   return (
     <CollapsibleDomainSection
-      title={pageUrl.host}
+      title={new URL(publishedOrigin).host}
```

Comment:
did you reverted this?

Reply:
shit, must be something wasn't updated before I started